static/src/stylesheets/base/_base.scss

```diff
@@ -19,8 +19,7 @@ body {
 
 
 @if $old-ie == false {
-    html,
-    body {
```

Comment:
We had to move this style because of this weird browser behaviour when combining `overflow-x` with `transform`: http://jsbin.com/yocanu/3/edit?html,css,output